### ISSUE TITLE
Downloading mooring data from THREADS (IMOS)

### ISSUE THREAD

**Issue by ribeiron** [oceanhackweek/OHW22-proj-plot_this_and_that#1]
- netcdf files in bulk;
- potentially find a way to download only the ones that have had some sort of manual QC;
- the netcdf files to be download are the QC'd ones.

**Comment by NickMortimer** [comment]
Yep we can also access the data directly from s3 it would also be good to add to the intake catalogue

**Comment by ribeiron** [comment]
The Downloading_one_file_threads.ipynb in /notebooks now loads one netcdf file from threads and plots it highlighting the full dataset against the good and bad data flags (I "hand picked" this file because I knew it had manually QC'ed data). @NickMortimer is working on a code that identifies the manual QC'ed ones within a folder (that is already sorted) and then downloads them in bulk (Nick's still figure out what is the best way to concatenate them).
